desc/compute/utils.py

```diff
@@ -362,6 +362,12 @@ def get_transforms(keys, obj, grid, jitable=False, **kwargs):
             transforms["rotmat"] = obj.rotmat
         elif c == "shift":
             transforms["shift"] = obj.shift
+        elif c == "potential":
```

Comment:
on one of my branches I changed this to be 
```python
elif hasattr(obj, c):
    transforms[c] = getattr(obj, c)
```
to avoid having all these different conditionals